### ISSUE TITLE
npm install throws permission error

### ISSUE THREAD

**Issue by dekadentno** [epicmaxco/vuestic-admin#60]
Hello and thank you for making this lovely template.
While following your instructions on the [wiki page](https://github.com/epicmaxco/vuestic-admin/wiki), I get the following error after hitting npm install:
```
root@happy-camper:/home/matej/Desktop/myproject# npm install
npm WARN deprecated babel-preset-es2015@6.24.1: 🙌  Thanks for using Babel: we recommend using babel-preset-env now: please read babeljs.io/env to update! 
npm WARN deprecated lodash.isarray@4.0.0: This package is deprecated. Use Array.isArray.
npm ERR! code 128
npm ERR! Command failed: /usr/bin/git clone -q git://github.com/amcharts/ammap3.git /root/.npm/_cacache/tmp/git-clone-a5908676
npm ERR! fatal: could not create leading directories of '/root/.npm/_cacache/tmp/git-clone-a5908676': Permission denied
npm ERR! 
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2017-12-14T12_52_59_048Z-debug.log
```
Googled it, found [this](https://docs.npmjs.com/getting-started/fixing-npm-permissions) but it isn't helping.
npm -v is 5.6.0
node -v is v8.9.3
I'm running Ubuntu 16.04.3 LTS.

**Comment by smartapant** [comment]
Hi @dekadentno 

I think you have some local problems, is npm throwing such errors for other projects?

**Comment by dekadentno** [comment]
Hello @smartapant 
I opened this issue because I was able to install other projects with the same steps but not this one. Because of that, I thought that maybe it wasn't a local problem.

After 2 days of trying to solve the problem with sudo, chown, chmod, unsafe-perm, updating node, updating/downgrading npm and so on, I was able to solve the problem 10 minutes ago with this command:
```
npm i -f
```
and now the project works like a charm.

Sorry for not noticing that this was indeed a local problem and opening this issue. I had it on 2 separate workstations and now I just hope this post will help someone in trouble.
